section3/exercises/arrays.js

```diff
@@ -24,42 +24,73 @@ console.log(animals[0]);
 
 // YOU DO: Write code below that will log the number of elements in array of
 // animals from above.
+var animals = ["Zebra", "Giraffe ", "Elephant"];
+
+console.log(animals.length);
 
 
 // YOU DO: Write code that will reassign the last item in the animals
 // array to "Gorilla"
+var animals = ["Zebra", "Giraffe", "Elephant"];
+animals.splice(2, 1, "Gorilla");
+console.log(animals);
 
+var animals = ["Zebra", "Giraffe", "Elephant"];
+animals[2] = "Gorilla"
+console.log(animals)
 
-// YOU DO: Write code that will add a new animal (type of your choice) to position 3.
 
+// YOU DO: Write code that will add a new animal (type of your choice) to position 3.
+var animals = ["Zebra", "Giraffe", "Gorilla"];
+animals[3] = "Penguin";
+console.log(animals)
 
 // YOU DO: Write code that will log the String "Elephant" in the animals array
+var animals = ["Zebra", "Giraffe", "Elephant", "Penguin"];
+console.log(animals[2])
+
 
 
 //-------------------
 // PART 2: Foods: Array Methods
 //-------------------
 
 // YOU DO: Declare a variable that will store an an array of at least 4 foods (strings)
-
+var foods = ["carrot", "beet", "plum", "bread"]
 
 // YOU DO: Write code below that will log the number of elements in the array of
 // foods from above.
+var foods = ["carrot", "beet", "plum", "bread"];
+console.log(foods.length)
 
 
 // YOU DO: Write code below that uses a method to add "broccoli" to the foods array and
 // log the changed array to verify "broccoli" has been added
+var foods = ["carrot", "beet", "plum", "bread"];
+foods[4] = "broccoli";
+console.log(foods)
 
 
 // YOU DO: Write code below that removes the last item of food from the foods array and
 // log the changed array to verify that item has been removed
+var foods = ["carrot", "beet", "plum", "bread", "broccoli"];
+foods.pop();
+console.log(foods)
+
 
 
-// YOU DO: Write code to add 3 new foods to the array. 
+// YOU DO: Write code to add 3 new foods to the array.
   // There are several ways to do this - choose whichever you'd like!
 // Then, log the changed array to verify the new items have been added
+var foods = ["carrot", "beet", "plum", "bread"];
+console.log(foods.unshift("potato", "kiwifruit", "pasta"))
```

Comment:
Another place where you don't want to call the method within a `console.log`. 

What you did on lines 77-78 model what you DO want 😊